help/de/Help_ch_Account-Actions.xml

```diff
@@ -437,6 +432,17 @@
       hierzu sind im Abschnitt <xref linkend="acct-types" /> beschrieben. Dies dient dazu, die
       Struktur der Kontenübersicht konsistent zu halten.
     </para>
+<!-- FIXME source: new tip -->
+    <tip>
+      <para>Zumindest wenn Sie planen, aus Ihren Daten offizielle Dokumente wie Steuererklärungen zu erstellen,
+        sollten Sie die gängigen Regeln Ihrer Gesetzgebung beachten. Fragen Sie im Zweifelsfall
+        Ihren Steuerberater. Für einige Länder wie AT, CH, DE haben Anwender Vorlagen
```

Comment:
Evtl. die beiden ersten Sätzen mit caution und den Rest als Tip kennzeichnen.

Und in den Backport noch:
```
<!-- Translators: For your country list search https://github.com/Gnucash/gnucash/tree/maint/data/accounts for the usual language+country codes 
[hm, irgendwas fehlt da noch, daß die nicht die privaten "common" templates lusten] -->

```

Reply:
ok.